mmv1/third_party/terraform/services/cloudrunv2/resource_cloud_run_v2_service_test.go.tmpl

```diff
@@ -1383,3 +1383,31 @@ resource "google_cloud_run_v2_service" "default" {
 `, context)
 }
 {{- end }}
+
+func testAccCloudRunV2Service_withInvokerIamDisabled(context map[string]interface{}) string {
```

Comment:
I notice you added this function but it's not referenced anywhere. Is there a reason for that? Did you mean to add it to an existing handwritten test?

Reply:
My bad; I was following a previous example; I now removed this, since it provides the same coverage than the VCR test.